test/reactive-handler.spec.ts

```diff
@@ -815,6 +881,36 @@ describe('ReactiveHandler', () => {
             expect(accessSpy).toHaveBeenCalledWith(value, '1');
         });
 
+        it('should handle setting length correctly', function() {
+            const target = new ReactiveMembrane();
+            const obj = {foo: ['bar', 'baz'] };
+            const property = target.getProxy(obj);
+
+            property.foo.length = 0;
+
+            expect(property.foo).toEqual([]);
+
+            property.foo.length = 0;
+
+            expect(property.foo).toEqual([]);
```

Comment:
```suggestion
```

Reply:
This duplication is deliberate. I'll add a comment.